test/e2e/apimachinery/resource_quota.go

```diff
@@ -110,7 +122,14 @@ var _ = SIGDescribe("ResourceQuota", func() {
 		framework.ExpectNoError(err)
 	})
 
-	ginkgo.It("should create a ResourceQuota and capture the life of a secret.", func() {
+	/*
+		Release: v1.16
+		Testname: ResourceQuota, object count quota, secret
+		Description: Create a ResourceQuota. Creation MUST be successful and its ResourceQuotaStatus MUST match to expected used and total allowed resource quota count within namespace.
+		Create a Secret. Its creation MUST be successful and resource usage count against the Secret object and resourceQuota object MUST be captured in ResourceQuotaStatus of the ResourceQuota.
+		Delete the Secret. Deletion MUST succeed and resource usage count against the Secret object MUST be released from ResourceQuotaStatus of the ResourceQuota.
+	*/
```

Comment:
Is this test no longer flaky?  For a very long time it was.  @kubernetes/sig-api-machinery-api-reviews am I recalling correctly?  Did jordan's fixes finally squash this?

Reply:
Seems not flaking.
https://k8s-testgrid.appspot.com/sig-release-master-blocking#gce-cos-master-default&include-filter-by-regex=should%20create%20a%20ResourceQuota%20and%20capture%20the%20life%20of%20a%20secret&graph-metrics=test-duration-minutes

Reply:
It would be useful to comment in the code below why the test need to wait for the number of secrets to stabilize

Reply:
It would be nice but I wouldn't block promoting on the lack of such a comment.  The test has been like this for a while.

ref @smarterclayton https://github.com/kubernetes/kubernetes/pull/40340

>On contended servers the service account controller can slow down,
leading to the count changing during a run. Wait up to 5s for the count
to stabilize, assuming that updates come at a consistent rate, and are
not held indefinitely.

Reply:
Added comment as well to bring visibility.( helpful for new contributor )